docs/faqs.md

```diff
@@ -6,4 +6,7 @@ title: FAQs
 # Frequently Asked Questions
 
 ### How Do I Generate the Scala Codecs? 
+Run the the `morphir gen` command with the -s flag
```

Comment:
Are you sure this is right? Shouldn't there be a specific option to enable codec generation?

Reply:
This has been updated and tested. The command is `morphir scala-gen -s' based on this PR https://github.com/finos/morphir-elm/pull/1073/files